package.json

```diff
@@ -32,23 +31,20 @@
     }
   },
   "devDependencies": {
-    "@11ty/eleventy": "0.12.1",
+    "@11ty/eleventy": "1.0.0",
     "@commitlint/cli": "16.2.3",
     "@commitlint/config-conventional": "16.2.1",
     "ava": "4.1.0",
-    "eslint-config-fluid": "2.0.0",
+    "eslint-config-fluid": "2.0.1",
     "fluid-lint-all": "1.1.5",
     "husky": "4.3.8",
     "infusion": "4.0.0",
-    "rimraf": "3.0.2",
-    "standard-version": "9.3.2"
+    "rimraf": "3.0.2"
   },
   "dependencies": {
-    "html-minifier": "4.0.0",
-    "markdown-it": "12.3.2",
-    "slugify": "1.6.5"
+    "html-minifier": "4.0.0"
   },
   "peerDependencies": {
-    "infusion": "4.x"
+    "infusion": "4.0.0"
```

Comment:
I think it's likely going to be better to keep a more lenient range or the peerDependency that accepted any stable release of Infusion 4. That way if we do put out any minor or patch releases, this won't necessarily need to be updated. 

see: [Using Peer Dependencies](https://nodejs.org/es/blog/npm/peer-dependencies/#using-peer-dependencies)